modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.tsx

```diff
@@ -14,6 +14,10 @@
 
 import ClayTooltip from '@clayui/tooltip';
 import {render, useTimeout} from '@liferay/frontend-js-react-web';
+
+// Hack to get around frontend-js-web not being in TS
+// @ts-ignore
```

Comment:
Yep, we may end up having to do a lot of this 😢, unless we start at the leaf nodes... (of which `frontend-js-web` is an obvious one).

Because `frontend-js-web` is so "core", we should probably try to port it, at least partially, sooner rather than later. Otherwise all the effectively `any` types that are going to percolate outwards from wherever it is use are going to weaken the guarantees provided by the type system, and probably be painful to clean up too.